neqo-transport/src/events.rs

```diff
@@ -126,6 +126,11 @@ impl ConnectionEvents {
         self.insert(ConnectionEvent::ZeroRttRejected);
     }
 
+    pub fn remove_stream_readable_events(&self, stream_id: StreamId) {
```

Comment:
Can we name this `recv_stream_complete` to match the existing `send_stream_complete` please?